client/tracing/src/lib.rs

```diff
@@ -196,3 +200,7 @@ fn send_telemetry(span_datum: SpanDatum) {
 		"time" => span_datum.overall_time.as_nanos(),
 	);
 }
+
+fn send_grafana(span_datum: SpanDatum) {
+	record_metrics!(span_datum.target => span_datum.overall_time.as_nanos());
```

Comment:
yeah, not sure the correct way to represent this for grafana, but seeing as we're limited to just a single `key => value`, we probably need to just concatenate `span_datum.target` and `span_datum.name`.

maybe `record_metrics!(format!("{}::{}", span_datum.target, span_datum.name) => span_datum.overall_time.as_nanos());` or something similar, if that works for grafana?